src/test/scala/com/amazon/deequ/anomalydetection/seasonal/HoltWintersTest.scala

```diff
@@ -207,6 +207,73 @@ class HoltWintersTest extends AnyWordSpec with Matchers {
       anomalies should have size 3
     }
   }
+
+  "work on hourly data with daily seasonality" in {
```

Comment:
Can you add a test for custom `seriesPeriodicity` as well? The two tests are testing the original constructor and not the new constructor.